src/test/java/com/google/api/generator/engine/writer/ImportWriterVisitorTest.java

```diff
@@ -255,6 +266,45 @@ public void importFromVaporAndConcreteReferences() {
             "import java.util.List;\n\n"));
   }
 
+  @Test
+  public void writeAnonymousClassExprImports(){
+    ConcreteReference stringListRef = ConcreteReference.builder().setClazz(List.class).setGenerics(Arrays.asList(ConcreteReference.withClazz(String.class))).build();    
+    ConcreteReference ref =
+        ConcreteReference.builder()
+            .setClazz(Function.class)
+            .setGenerics(
+                Arrays.asList(
+                    stringListRef,
+                    ConcreteReference.withClazz(Integer.class)))
```

Comment:
Similar comment as with the String class above, let's use something that needs to be imported.